.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Can we do labels which are depending on the issue type, like "bug" or "feature"?

Reply:
It is possible to define multiple issue templates for different purposes which require users to select an issue type (like [this](https://github.com/thephez/exp/issues/new/choose)). I was trying to keep it simple/generic. We could have 2 options though and still keep it generic enough to use on most repos.

Reply:
@shumkov Opened alternative PR #47 with 2 templates that include labels.

Reply:
Awesome! We need more guys to check it before we merge it.